core/inject.js

```diff
@@ -164,7 +168,9 @@ Blockly.createDom_ = function(container) {
   goog.ui.Component.setDefaultRightToLeft(Blockly.RTL);
 
   // Load CSS.
-  Blockly.Css.inject();
+  if (!opt_external_css) {
+    Blockly.Css.inject();
```

Comment:
Not injecting CSS is a problem since Blockly dynamically modifies the CSS to change the cursor.  This may be a new behaviour that the ancient google3 copy did not have.

I've created a patch that injects the required cursor stub, even if the bulk of the CSS is flagged off:  https://github.com/google/blockly/commit/f8365e9eba2851f4ddd223cf62676fa91eeadd3d